src/sdk/vscode/README.md

```diff
@@ -39,3 +39,11 @@ All requirements will be checked on each command invocation
 - `iotea.rule.or` + `Strg + Space` > Create new Or Rule
 - `iotea.rule.op` + `Strg + Space` > Create a new Rule with an Operation Constraint
 - `iotea.rule.change` + `Strg + Space` > Create a new Rule with a Change Constraint
+
+## Troubleshooting
+
+- _"Docker version check fails"_<br>
+  Make sure you have the latest Docker version installed, since `docker version --format=json` is used to retrieve the version. In older versions the string "json" was returned
+- _"Python version check fails"_<br>
+  You can specify the Python interpreter and the pip module in the settings. It defaults to `python3` and `pip3` for standalone installations. If you are running a virtual environment setup based on Anaconda, you have to use `python` and `pip`. On Linux-Systems it can be different. Please check in a terminal, which command to use and update the settings accordingly.
```

Comment:
The python _module_ for pip is not called `pip3` on any platform I've tested.

The default value for pip _module_ should be changed to `pip` which would then work with the default installations of python on both windows and linux.

Reply:
The checking is done like this (slightly pseudo-codified for comprehension):

`${iotea.terminal.python} -m ${iotea.terminal.pip} --version`

so, it's using the configured python executable to invoke the pip _module_ specified in `iotea.terminal.pip`.

If it was looking for the stand-alone pip, then the default value of `pip3` would make sense (at least on some systems).